plutus-pab/tx-inject/start-all-servers.sh

```diff
@@ -1,5 +1,5 @@
 #!/bin/bash
-rm -f ./primary.db ./node-server.sock
+rm -f ./primary.db
 cabal build all
```

Comment:
can we change this to `cabal build plutus-pab`? There are quite a few build targets in `all` that we don't need